CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## main
 
+### Breaking changes
```

Comment:
Mind including specific directions for migrating the breaking `animation` change in prod here?